jerry-core/ecma/base/ecma-gc.c

```diff
@@ -427,6 +430,13 @@ ecma_gc_sweep (ecma_object_t *object_p) /**< object to free */
           ecma_free_value (ext_object_p->u.class_prop.value);
           break;
         }
+        case LIT_MAGIC_STRING_ARRAY_BUFFER_UL:
+        {
+          arraybuffer_length = ecma_get_uint32_from_value (ext_object_p->u.class_prop.value);
+          is_contain_arraybuffer = true;
+          ecma_free_value (ext_object_p->u.class_prop.value);
+          break;
```

Comment:
Is there a built-in array buffer? If not, we can just free the object here. Furthermore we need guards, since array-buffer might be disabled at compile time.